cmd/fyne/internal/commands/mock_runner_test.go

```diff
@@ -39,14 +39,12 @@ type testCommandCall struct {
 
 func (t *testCommandCall) runOutput(args ...string) ([]byte, error) {
 	// Check that we have less than the expected number of call
-	assert.Less(t.t, t.index, len(t.calls))
+	require.Less(t.t, t.index, len(t.calls))
 	// Check that we have the expected number of parameters for this call
-	assert.Equal(t.t, len(t.calls[t.index].args), len(args))
+	require.Equal(t.t, len(t.calls[t.index].args), len(args))
 	// Check that each argument match our expectation
-	if len(t.calls[t.index].args) == len(args) {
-		for index, value := range args {
-			assert.Equal(t.t, t.calls[t.index].args[index], value)
-		}
+	for index, value := range args {
+		require.Equal(t.t, t.calls[t.index].args[index], value)
```

Comment:
Good use of `require`, but in this case should it not still be `assert` so we continue to test the other items? Best to check as much as you can without crashing.